packages/server/src/automations/triggers.ts

```diff
@@ -171,3 +203,28 @@ export async function rebootTrigger() {
     })
   }
 }
+
+async function checkTriggerFilters(
+  automation: Automation,
+  event: { row: Row; oldRow: Row }
+): Promise<boolean> {
+  const trigger = automation.definition.trigger
+  const filters = trigger?.inputs?.filters
+  const tableId = trigger?.inputs?.tableId
+
+  if (!filters) {
+    return true // No filters or tableId, so trigger by default
```

Comment:
Comment seems to be out of sync with code, `tableId` is not checked. I'd probably just ditch this comment, the comment isn't really telling us anything the code isn't.